go.mod

```diff
@@ -14,7 +14,19 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/valyala/fasthttp v1.34.0
 	golang.org/x/net v0.0.0-20220225172249-27dd8689420f
-	google.golang.org/genproto v0.0.0-20210114201628-6edceaf6022f // indirect
```

Comment:
Everything on this line and below was auto generated by "go mod tidy". That includes creating a separate required block.

Reply:
I was curious, and it appears that since go 1.17, separate "require" blocks is intended design.
https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17/69433550